source/16-6-2_Outcomes.markdown

```diff
@@ -5,12 +5,30 @@
 
 Adding, editing and deleting {{outcome}}s works in the same way as for other lists, except that you have a main {{outcome}} category and {{outcome}} measures within each category. 
 
-You can have as many categories as you need, and each can have any number of {{outcome}} measures in it. For practical reasons we'd suggest no more than ten measures in each category. 
+You can have as many categories as you need, and each can have any number of {{outcome}} measures in it. For practical reasons we'd generally suggest no more than ten measures in each category. 
 
-If you are using an established {{outcome}} framework (e.g. Core, WEBWMS, Outcome Stars) these can be imported directly into your system without having to enter all the details, and in many cases sub totals and totals, and descriptions for individual scores, will also be set up for you.  Please contact the Lamplight help desk if you'd like to do this before entering them manually into your system.
+If you wish to you can add "descriptors" for each possible score in each measure. These will help your team and your 
+clients to understand what each score means. For example, if you have a scale of 1 - 5, you might want to add a 
+descriptor for each score, such as "1 = very unhappy" and "5 = very happy".
+
+If you are using an established {{outcome}} framework (e.g. Core, WEBWMS, Outcome Stars) these can be imported 
+directly into your system without having to enter all the details, and in many cases sub-totals and totals, and 
```

Comment:
without **you** having to enter all the details
**ADD TEXT**
all the details**, and in** many cases
all the **details.  In** many cases